### PR TITLE
Dev/progress bar svgs

### PR DESCRIPTION
**Pull request by batteries76**
This one is just using the SVGs for the progress bar. 
It's a hacky solution, but also hews more closely to the designs and is quick.
Requires a package that brings in the SVGs as React components.
Not sure of the plus minus between the two.

### CODE REVIEW

**Comment by hqtan** [comment]
Hi @batteries76 ,
thx for this PR! I'll check this out locally to have a look at it, and provide some feedback this weekend. :bow:

**Comment by hqtan** [comment]
taking screenshots of form pages for easy reference/comparison:
- page 1 (`pages/form/1.js`)
  image
- page 2 (`pages/form/2.js`)
  image
- page 3 (`pages/form/3.js`) when a user selected 'Office / Onsite' for one or more days in page 2
  image
- page 3 (`pages/form/3.js`) when a user didn't select 'Office / Onsite' for any day in page 2
  image
- page 4 (`pages/form/4.js`)
  image
- page 5 (`pages/form/5.js`)
image

**Comment by hqtan** [comment]
Hi @batteries76 ,

I saw 2 issues that require attention:
1. there is one case in page 3 (pages/form/3.js) where the progress bar does not appear. see the screenshots above. N.B: page 3 renders differently depending on the responses given in page 2.
2. the page title on page 5 (pages/form/5.js) below progress bar shows 'Question 4/5', instead of `Question 5/5'. (This was a pre-existing issue :sweat_smile: )

File: pages/form/5.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ export default function Question5() {
 
   return (
     <Layout>
+      <Q5Progress />
       <Heading>Question 4/5</Heading>
```
Comment (hqtan) [inline pages/form/5.js:18]:
```suggestion
      <Heading>Question 5/5</Heading>
```

File: pages/form/3.js [hunk 2]
```diff
@@ -26,6 +27,7 @@ export default function Question3() {
   if (travelDays.length < 1) {
     return (
       <Layout>
+        <Q3Progress />
```
Comment (hqtan) [inline pages/form/3.js:30]: I think this section requires a bit of refactoring.
the `<Q3Progress />` component needs to be moved outside of the `if` block, but remain within `<Layout />` block.

Lemme know if you need any help / suggestions with this.

**Review by hqtan — CHANGES_REQUESTED** [review]

File: pages/form/3.js (hunk 2 above)
Comment (batteries76) [reply pages/form/3.js:30]: Should the progress bar appear on the page regardless of which path the app takes @hqtan?

**Comment by batteries76** [comment]
Actually, I've only just noticed that the cloud's face changes as it moves (lack of attention to detail on my part). But that to me is really the nail in the coffin of other solutions. I would recommend we use the SVGs and keep going. Later we can revisit it. 
What do you think @hqtan?

File: pages/form/3.js (hunk 2 above)
Comment (hqtan) [reply pages/form/3.js:30]: Yups! progress bar should appear regardless of which path the form takes 👍

Comment (batteries76) [reply pages/form/3.js:30]: Ok cool. I think it needs to be in both the if statement and the other return. I have changed that my end and will update.

**Review by hqtan — APPROVED** [review]

**Comment by hqtan** [comment]
Thx for the changes @batteries76 !

One other thing: I'm gonna remove the `yarn.lock` file that you committed into the PR k? We already have a `package-lock.json` file in `main` branch; no need to commit another lock file...

**Pull request merged**